Framework/PythonInterface/plugins/algorithms/SetDetScale.py

```diff
@@ -51,6 +56,14 @@ def PyExec(self):
             comp, value = component.split(":")
```

Comment:
I would expect that if both the list and the file are given, the list should overwrite the scales given in the file. 
So I would move this loop after the loop over the lines in the file.